localisation/00_zuwc_united_localisation.csv

```diff
@@ -3016,11 +3012,21 @@ EVTOPTB_WCADD_1100;I'll pay [target_crystal_owner.GetHerHim] a visit!;;Ich werde
+DYN5_DESC;With a score of $SCORE$ you$FAILED$ beat House Sunstrider, the first known member being Dath'Remar. Born into wealth and power, he was obsessed with magic, causing the night elves to exile him and his kin after the War of the Ancients and thus he led his people across the sea, founding Quel'Thalas and the Sunwell. His descendant, Anasterian ruled for around 2,800 years, as well as during the Troll Wars. Anasterian's death caused the downward spiral of the dynasty. His son, Kael'thas, took up the mantle of leadership and attempted to salvage what was left of the Kingdom after the Scourge invasion.;;Mit einem Punktestand von $SCORE$ ist es Euch$FAILED$ gelungen, das Haus Sunstrider, dessen erstes bekanntes Mitglied Dath'Remar war, zu �bertreffen. Geboren in Reichtum und Macht, war er von Magie besessen und brachte die Nachtelfen dazu, ihn und seine Sippe nach dem Krieg der Ahnen ins Exil zu verbannen. So f�hrte er sein Volk �ber das Meer und gr�ndete Quel'Thalas und den Sonnenbrunnen. Sein Nachfahre, Anasterian, herrschte rund 2.800 Jahre lang, darunter auch w�hrend der Trollkriege. Anasterians Tod verursachte die Abw�rtsspirale der Dynastie. Sein Sohn, Kael'thas, nahm den Mantel der F�hrung an und versuchte, das zu retten, was nach der Invasion der Gei�el vom K�nigreich �brig geblieben war.;;;;;;;;;;;x
+DYN6_DESC;With a score of $SCORE$ you$FAILED$ beat house Anvilmar, the descendants of the last High King of all dwarves who ruled the kingdom with justice and wisdom, Modimus Anvilmar. As their society grew too large within the confines of their great mountain, three powerful factions rose among their people: the Bronzebeard, Wildhammer and Dark Iron clans. This divide eventually led to the downfall of the Anvilmar dynasty. When Modimus passed away of old age, his son would not have enough time to be officially coronated before the dwarven civil war had broken out between the three clans.;;Mit einem Punktestand von $SCORE$ ist es Euch$FAILED$ gelungen, das Haus Anvilmar, die Nachkommen des letzten Hochk�nigs aller Zwerge, der das K�nigreich mit Gerechtigkeit und Weisheit regierte, Modimus Anvilmar, zu �bertreffen. Als ihre Gesellschaft innerhalb der Grenzen ihres gro�en Berges zu gro� wurde, erhoben sich drei m�chtige Fraktionen in ihrem Volk: der Bronzebeard-, der Wildhammer- und der Dunkeleisen-Clan. Diese Spaltung f�hrte schlie�lich zum Untergang der Anvilmar-Dynastie. Als Modimus an Altersschw�che starb, hatte sein Sohn nicht genug Zeit, offiziell gekr�nt zu werden, bevor der B�rgerkrieg der Zwerge zwischen den drei Clans ausbrach.;;;;;;;;;;;x
```

Comment:
Standardizing capitalized House in `beat House xxx`
```suggestion
DYN6_DESC;With a score of $SCORE$ you$FAILED$ beat House Anvilmar, the descendants of the last High King of all dwarves who ruled the kingdom with justice and wisdom, Modimus Anvilmar. As their society grew too large within the confines of their great mountain, three powerful factions rose among their people: the Bronzebeard, Wildhammer and Dark Iron clans. This divide eventually led to the downfall of the Anvilmar dynasty. When Modimus passed away of old age, his son would not have enough time to be officially coronated before the dwarven civil war had broken out between the three clans.;;Mit einem Punktestand von $SCORE$ ist es Euch$FAILED$ gelungen, das Haus Anvilmar, die Nachkommen des letzten Hochk�nigs aller Zwerge, der das K�nigreich mit Gerechtigkeit und Weisheit regierte, Modimus Anvilmar, zu �bertreffen. Als ihre Gesellschaft innerhalb der Grenzen ihres gro�en Berges zu gro� wurde, erhoben sich drei m�chtige Fraktionen in ihrem Volk: der Bronzebeard-, der Wildhammer- und der Dunkeleisen-Clan. Diese Spaltung f�hrte schlie�lich zum Untergang der Anvilmar-Dynastie. Als Modimus an Altersschw�che starb, hatte sein Sohn nicht genug Zeit, offiziell gekr�nt zu werden, bevor der B�rgerkrieg der Zwerge zwischen den drei Clans ausbrach.;;;;;;;;;;;x
```

Reply:
thx